pkg/runtime/cli.go

```diff
@@ -26,7 +26,7 @@ func FromFlags() (*DaprRuntime, error) {
 	appProtocol := flag.String("protocol", string(HTTPProtocol), "Protocol for the application: gRPC or http")
 	componentsPath := flag.String("components-path", DefaultComponentsPath, "Path for components directory. Standalone mode only")
 	config := flag.String("config", "", "Path to config file, or name of a configuration object")
-	daprID := flag.String("dapr-id", "", "A unique ID for Dapr. Used for Service Discovery and state")
+	appID := flag.String("app-id", "", "A unique ID for Dapr. Used for Service Discovery and state")
```

Comment:
Should we change the usage description as well?

Reply:
No, this is still descriptive of what it means for Dapr.

Reply:
Isn't it an ID for the application? [Reference](https://github.com/dapr/cli/blob/master/docs/reference/dapr-run.md)

Reply:
Well, it's an ID for both. They are one in the same.